.github/workflows/manually_compile_check_image.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: .
-          file: ./docker/Dockerfile-compile-check-ubuntu18.04
+          file: ./docker/Dockerfile-compilation-testing-ubuntu18.04
           platforms: linux/amd64
           push: true
           tags: inclavarecontainers/test:compile-check-ubuntu18.04
```

Comment:
Please change tag to `compilation-check-ubuntu18.04` to be consistent with the tag in `pr_basic_compilation_check.yml`